.github/workflows/ci.yml

```diff
@@ -17,6 +17,8 @@ jobs:
   build-libs-linux:
     name: build-libs-linux-${{ matrix.arch }} 😁 (${{ matrix.name }})
     runs-on: ${{ (matrix.arch == 'arm64' && 'github-arm64-2c-8gb') || 'ubuntu-22.04' }}
+    container:
```

Comment:
At this point, do we even need the container? I mean, we already run on `ubuntu-22.04` on x86 and arm64.

Reply:
I'd remove the `container` directive.
Also, remember to add `sudo` before install deps steps